src/app/search/SearchFilter.tsx

```diff
@@ -20,13 +21,10 @@ import { StoreServiceType } from '../inventory/store-types';
 import { loadingTracker } from '../shell/loading-tracker';
 import SearchFilterInput from './SearchFilterInput';
 import { showNotification } from '../notifications/notifications';
+import NotificationButton from '../notifications/NotificationButton';
 import { CompareService } from '../compare/compare.service';
 
 const bulkItemTags = Array.from(itemTags) as any[];
-// t('Tags.TagItems')
-// t('Tags.ClearTag')
-// t('Tags.LockAll')
-// t('Tags.UnlockAll')
```

Comment:
Gotta keep this or the i18next cleaner will remove the strings

Reply:
:white: